parallel_tests.gemspec

```diff
@@ -10,5 +10,6 @@ Gem::Specification.new name, ParallelTests::VERSION do |s|
   s.license = "MIT"
   s.executables = ["parallel_spinach", "parallel_cucumber", "parallel_rspec", "parallel_test"]
   s.add_runtime_dependency "parallel"
+  s.add_runtime_dependency "sys-proctable"
```

Comment:
prefer less dependencies and this only working on mac/unix ... don't need that extra dependency for the frindge usecase ...